styles/main/style.css

```diff
@@ -364,21 +365,23 @@ html {
 .contacts__box{
   position: relative;
 }
-.contacts__box:before{
-  content: "Hello";
+.background__text:before{
   position: absolute;
-  top: 50%;
   left: 50%;
   transform: translate(-50%, -50%);
-  color: var(--contacts-light-gray);
-  font-size: 550px;
   letter-spacing: 0.13em;
   text-align: center;
   z-index: -1;
-  overflow: hidden;
   max-width: 100%;
   line-height: 1;
 }
```

Comment:
.background__text is a reusable class, so it shouldn't be in /* Contacts section */